tests/download.spec.ts

```diff
@@ -378,7 +378,7 @@ it.describe('download event', () => {
       page.context().close(),
     ]);
     expect(downloadPath).toBe(null);
-    expect(saveError.message).toContain('File deleted upon browser context closure.');
+    expect(saveError.message).toContain('File not found on disk. Check download.failure() for details.');
```

Comment:
I don't think this is the correct error message, but if it is it needs to be conditional on the browser version here.

Reply:
The message has changed because we now map  download state `INTERRUPTED` to `cancelled` on [chrome side](https://chromium-review.googlesource.com/c/chromium/src/+/2804358/8/content/browser/devtools/protocol/page_handler.cc#592) (previously that state was incorrectly mapped to `InProgress`). Due to this change we know receive download cancelled  event during context closure and that is the first error dispatched to the download artifact in playwright. It looks legit to me, I'll add a condition based on browser version as you suggest.